File: app/src/main/java/it/paranoidsquirrels/idleguildmaster/storage/data/items/instances/BlackIronIngot.java
package it.paranoidsquirrels.idleguildmaster.storage.data.items.instances;

import it.paranoidsquirrels.idleguildmaster.R;
import it.paranoidsquirrels.idleguildmaster.storage.data.items.Item;

/* JADX INFO: loaded from: classes3.dex */
public class BlackIronIngot extends Item {
    @Override // it.paranoidsquirrels.idleguildmaster.storage.data.items.Item
    protected void configureProperties() {
        this.idName = R.string.item_black_iron_ingot_name;
        this.idDescription = R.string.item_black_iron_ingot_description;
        this.idImage = R.drawable.black_iron_ingot;
        this.price = 14L;
    }
}

File: app/src/main/kotlin/it/paranoidsquirrels/idleguildmaster/storage/data/items/instances/BlackIronIngot.kt
package it.paranoidsquirrels.idleguildmaster.storage.data.items.instances

import it.paranoidsquirrels.idleguildmaster.R
import it.paranoidsquirrels.idleguildmaster.storage.data.items.Item

class BlackIronIngot : Item() {
    override fun configureProperties() {
        idName = R.string.item_black_iron_ingot_name
        idDescription = R.string.item_black_iron_ingot_description
        idImage = R.drawable.black_iron_ingot
        price = 14L
    }
}
